modules/osdk-csv-manual-annotations.adoc

```diff
@@ -18,7 +18,7 @@ The following table lists Operator metadata annotations that can be manually def
 |Provide custom resource definition (CRD) templates with a minimum set of configuration. Compatible UIs pre-fill this template for users to further customize.
 
 |`operatorframework.io/initialization-resource`
-|Specify a single required custom resource that must be created at the time that the Operator is installed. Must include a template that contains a complete YAML definition.
+|Specifies a single required custom resource by adding `operatorframework.io/initialization-resource` annotation to the cluster service version (CSV) during Operator installation. User is then prompted to create the custom resource through a template that is provided in the CSV.  Must include a template that contains a complete YAML definition.
```

Comment:
For parallelism with the rest of the table, should `Specifies` be `Specify`? Also - try to avoid the passive tense in the second sentence. I am not familiar with this part of the product/doc set to provide a suggestion.

Reply:
yes makes sense on the parallelism

Reply:
@mjpytlak thanks , ptal